Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM debian:stretch
+
+COPY . /usr/src/kafkacat
+
+RUN runDeps=''; \
```

Comment:
You could drop `runDeps` since it's empty, also, you could avoid setting `-ex` and use ` && ` between commands.